Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN bundle exec middleman build --clean
 #
 # Stage 3: build dev center
 #
-FROM ubuntu:21.04 as dev-center-builder
+FROM ubuntu:20.10 as dev-center-builder
```

Comment:
do we not stick to LTS releases?

Reply:
That's a great point. I jumped the gun on this the last time I updated the Dockerfile. I'll update to 20.04. [Looks like](https://easylinuxtipsproject.blogspot.com/p/ubuntu.html#:~:text=Ubuntu%2020.04%20is%20an%20LTS,but%20Ubuntu%2020.04%20is%20excellent.&text=Intermediate%20versions%20(interim%20releases)%20are,LTS%20is%20simply%20much%20better.) 20.04 is the latest LTS version.